tests/cpu_bugs/login_console.pm

```diff
@@ -0,0 +1,45 @@
+# SUSE's openQA tests
+#
+# Copyright © 2018 SUSE LLC
```

Comment:
2019?

Reply:
might be this module also need remove, because login_console is everywhere in our code.